src/styles/hero-header.styl

```diff
@@ -35,36 +32,40 @@ whiteOnWhiteShadow = drop-shadow(0 1px 1px rgba(50, 54, 63, .32)) drop-shadow(0
         width 8rem
         height 8rem
         border-radius 50%
-        border 4px solid var(--white)
+        border 2px solid var(--white)
         box-shadow 0 2px 16px 0 textShadowHeavy
         margin-bottom 1rem
 
         +tiny-screen()
-            width 4rem
-            height 4rem
-            border-width 2px
+            width 4.5rem
+            height 4.5rem
             margin-bottom .25rem
 
     .hero-title
-        font-size 2rem
-        font-weight bold
-        line-height 1.25
         color var(--primaryContrastTextColor)
-        margin 0
+        font-size 3rem
+        font-weight bold
+        letter-spacing -.8px
+        line-height 1.08333
```

Comment:
Is it wanted that those 2 values are in `px` and not `rem` ?

Reply:
1.0833 doesn't exist https://docs.cozy.io/cozy-ui/styleguide/section-utilities.html#kssref-utilities-text-line-height

Reply:
fixed using rem() utility